model_utils/tracker.py

```diff
@@ -76,19 +79,31 @@ def __get__(field, instance, owner):
                     self.saved_data[field.field_name] = deepcopy(value)
                 return data[field.field_name]
 
-        for field in self.fields:
-            field_obj = self.instance.__class__.__dict__.get(field)
-            if isinstance(field_obj, DeferredAttribute):
-                self.instance._deferred_fields.append(field)
-
-                # Django 1.4
-                model = None
-                if hasattr(field_obj, 'model_ref'):
-                    model = field_obj.model_ref()
-
+        if django.VERSION >= (1, 8):
+            self.instance._deferred_fields = self.instance.get_deferred_fields()
+            for field in self.instance._deferred_fields:
+                if django.VERSION >= (1, 10):
+                    field_obj = getattr(self.instance.__class__, field)
```

Comment:
It doesn't.
[This change](https://github.com/django/django/commit/7f51876f99851fdc3fef63aecdfbcffa199c26b9#diff-1e7fc0d7d1b36358e371fab97bd1ddb1R99) is what enabled it.